moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -1029,14 +1029,16 @@ bool PlanningSceneMonitor::waitForCurrentRobotState(const rclcpp::Time& t, doubl
   // Sometimes there is no state monitor. In this case state updates are received as part of scene updates only.
   // However, scene updates are only published if the robot actually moves. Hence we need a timeout!
   // As publishing planning scene updates is throttled (2Hz by default), a 1s timeout is a suitable default.
+  auto start = node_->get_clock()->now();
+  auto timeout = rclcpp::Duration(wait_time);
   boost::shared_lock<boost::shared_mutex> lock(scene_update_mutex_);
   rclcpp::Time prev_robot_motion_time = last_robot_motion_time_;
   while (last_robot_motion_time_ < t &&  // Wait until the state update actually reaches the scene.
-         timeout > std::chrono::seconds())
+         timeout > rclcpp::Duration(0))
```

Comment:
CI is failing because of this line I believe.  You should be able to get it to pass with this:

```suggestion
         timeout > rclcpp::Duration::from_seconds(0))
```

or

```suggestion
         timeout > rclcpp::Duration(0, 0))
```